### PR TITLE
146 cons german profilinfo page

### PR DESCRIPTION
**Pull request by radscheit**

### CODE REVIEW

File: config/locales/de.devise.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 de:
   devise:
+    messages:
+      leave_blank_if_you_don_t_want_to_change_it: "Offen lassen, falls du dein Passwort jetzt nicht verändern möchtest."
```
Comment (Blaidd-Drwg) [inline config/locales/de.devise.yml]: Why "offen" and not "leer"? This would have confused me.

File: config/locales/de.devise.yml [hunk 2]
```diff
@@ -56,5 +59,5 @@ de:
       not_found: "nicht gefunden"
       not_locked: "ist nicht gesperrt"
       not_saved:
-        one: "Konnte %{resource} nicht speichern: ein Fehler."
-        other: "Konnte %{resource} nicht speichern: %{count} Fehler."
\ No newline at end of file
+        one: "<span style='text-transform:uppercase'>%{resource}</span> konnte nicht gespeichert werden: ein Fehler. Bitte korrigiere deine Eingabe um fortzufahren."
+        other: "<span style='text-transform:uppercase'>%{resource}</span> konnte nicht gespeichert werden: %{count} Fehler. Bitte korrigiere deine Eingaben um fortzufahren."
```
Comment (Blaidd-Drwg) [inline config/locales/de.devise.yml]: Have you tested this in your browser? I think I18n escapes HTML tags by default, in which case this would not work.

Comment (Blaidd-Drwg) [inline config/locales/de.devise.yml]: Also missing a comma before the "um" in both lines

File: spec/features/user_spec.rb [hunk 3]
```diff
@@ -35,20 +35,20 @@
     expect(page).to have_css(".alert-success")
   end
 
-  scenario "User changes name on user settings page" do
+  scenario "User is not able to change email on user settings page" do
```
Comment (Blaidd-Drwg) [inline spec/features/user_spec.rb]: Why "is not"?

File: spec/views/navbar_spec.rb [hunk 4]
```diff
@@ -21,8 +21,8 @@
       render template: 'application/index', layout: 'layouts/application'
     end
 
-    it "shows Profilinfo, Mein Profil anlegen, Meine Bewerbungen, Ausloggen in the dropdown" do
-      expect(rendered).to have_css(".nav .dropdown-menu a", text: 'Profilinfo')
+    it "shows Einstellungen, Mein Profil anlegen, Meine Bewerbungen, Ausloggen in the dropdown" do
+      expect(rendered).to have_css(".nav .dropdown-menu a", text: 'Einstellungen')
       expect(rendered).to have_css(".nav .dropdown-menu a", text: 'Mein Profil anlegen')
       expect(rendered).to have_css(".nav .dropdown-menu a", text: 'Meine Bewerbungen')
       expect(rendered).to have_css(".nav .dropdown-menu a", text: 'Ausloggen')
```
Comment (Blaidd-Drwg) [inline spec/views/navbar_spec.rb]: These should all use I18n

**Review by Blaidd-Drwg — CHANGES_REQUESTED** [review]

File: config/locales/de.devise.yml (hunk 1 above)
Comment (radscheit) [inline config/locales/de.devise.yml]: Geschmacksfrage, aber mir nicht wichtig. Ich schreibe leer.

File: spec/views/navbar_spec.rb (hunk 4 above)
Comment (radscheit) [inline spec/views/navbar_spec.rb]: Good idea.

**Comment by Blaidd-Drwg** [comment]
Looks good to me. Could you undo your changes to `config/locales/de.default.yml`, since those are already part of #225 and are being reviewed there? When that is done, the PR can be merged :)

**Review by Blaidd-Drwg — APPROVED** [review]

**Pull request merged**